ui/components/app/assets/defi-list/defi-list.tsx

```diff
@@ -145,10 +145,7 @@ export default function DefiList({ onClick }: DefiListProps) {
           );
         })
       ) : (
-        <DeFiEmptyStateMessage
-          primaryText={t('noDeFiPositions')}
-          secondaryText={t('protocolNotSupported')}
-        />
+        <DeFiEmptyStateMessage />
```

Comment:
Removing removed props and moving locales and other logic into the `DeFiEmptyStateMessage`